### PR TITLE
Update calypso button styles

### LINKED ISSUE #89: Create Your First Product Purple Hover State
On a fresh install with no products created, the large "Create Your First Product" button still has a purple hover state:

image

### LINKED ISSUE #102: Match calypso compact buttons
There are a few minor changes that need to be made to match some of the dashboard buttons to Calypso's ( https://wpcalypso.wordpress.com/devdocs/design/button )

* Focus color still seems to be purple in some places
* Box shadow
* Hover background
* Hover box-shadow

#### Focus
screen shot 2018-10-29 at 9 02 26 am

#### Hover
screen shot 2018-10-29 at 9 06 11 am

### LINKED ISSUE #113: Subscriptions: update button turns purple after update
The update button on the Subscriptions screen turns purple and has different font weight and width after the update:

update

The design is also no aligned with the calypso buttons, but will create a specific issue for that since it impacts all screens.

### PR DESCRIPTION
**Pull request by joshuatf**
Fixes #102 fixes #89 fixes #113 

This PR enforces calypso button styles for wp core buttons and buttons in extensions, including hover and focus styles.

### Screenshots
screen shot 2018-10-30 at 1 52 51 pm
screen shot 2018-10-30 at 1 53 38 pm
screen shot 2018-10-30 at 9 27 41 am

### Testing
1.  Enable Calypsoify using the query param `calypsoify=1` in the URL.
2.  Navigate through all the Calypsoified areas and check that buttons match the [Calypso button styles](https://wpcalypso.wordpress.com/devdocs/design/button).  Specifically, here are a few places to check:
* `/wp-admin/admin.php?page=wc-setup`
* `/wp-admin/admin.php?page=wc-addons`
* `/wp-admin/edit.php?post_type=shop_order`
3.  Check that hover, focus, and active states are styled properly.

### CODE REVIEW

File: wc-calypso-bridge.php [hunk 1]
```diff
@@ -36,7 +36,7 @@
 // TODO Pick a better option name.
 // We can set this during store setup/provisioning so they get the right code loaded.
 // @codingStandardsIgnoreLine
-// update_option( 'is-atomic-ecommerce', true );
+update_option( 'is-atomic-ecommerce', true );
```
Comment (timmyc) [inline wc-calypso-bridge.php]: Was this change intentional, or should we back this out?

Comment (joshuatf) [reply wc-calypso-bridge.php]: Whoops- unintentional.  Fixed up.
